src/components/Dropdown/styles.scss

```
@@ -0,0 +1,82 @@
+.container{
+  width: 90%;
+  display: flex;
+  flex-direction: column;
+}
+
+.dropdown{
+    all: unset;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    height: 56px;
+    padding: 16px;
+    margin-bottom:8px;
+    background: #F5F5F5;
+    border-radius: 8px;
+    box-sizing: border-box;
+    &:active{
+        background: #BBBBBB;
+    }
+    &:hover{
+        background: #E6E6E6;
+    }
+}
+
+.itemsContainer{
+    all: unset;
+    z-index: 10;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+    background: #F5F5F5;
+    opacity: 0;
+    width: 100%;
+    height: 100%;
+    position: absolute ;
```

You should use a formatter for css files.
```suggestion
    position: absolute;
```